api/model/src/main/java/org/projectnessie/model/ContentKey.java

```diff
@@ -171,10 +172,12 @@ protected void validate() {
         throw new IllegalArgumentException(
             String.format("Content key '%s' must not contain a null element.", elements));
       }
-      if (e.contains(ZERO_BYTE_STRING) || e.contains(GROUP_SEPARATOR_STRING)) {
```

Comment:
Wonder whether we should forbid all chars < 0x20 - guess, those don't make any sense at all.

Reply:
updated